src/webgpu/shader/execution/memory_model/litmus.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+
+export const description = `
+A suite of tests for properties of the WebGPU memory model involving two memory locations.
```

Comment:
Quibble with the name "litmus".  In a real sense I think all the tests in this suite are litmus tests.
Suggest renaming to "weak.spec.ts" ?